openshift-gitops/overlays/nerc-ocp-infra/patches/argocds/openshift-gitops.yaml

```diff
@@ -18,3 +18,12 @@ spec:
       - "ManagedClusterInfo"
       clusters:
       - "*"
+  rbac:
+    policy: |
+      p, role:operator, applications, sync, */*, allow
+      p, role:operator, applications, update, */*, allow
+
+      g, system:cluster-admins, role:admin
+      g, cluster-admins, role:admin
+      g, nerc-ops, role:operator
```

Comment:
the linked argocd document examples have this as `your-github-org:your-team` but I guess since we are syncing the groups we don't specify the github org? Just trying to understand. 

Also what does the `g` mean? The argocd document used this in examples but didn't explain what it is.

Reply:
> the linked argocd document examples have this as your-github-org:your-team but I guess since we are syncing the groups we don't specify the github org?

Right, we're simply referencing openshift groups in this policy.

the linked argocd document examples have this as your-github-org:your-team but I guess since we are syncing the groups we don't specify the github org? Just trying to understand.

> Also what does the g mean? The argocd document used this in examples but didn't explain what it is.

I noticed that lack of documentation as well!  I've decided the `p` means `policy` and `g` means `group`. In fact, with a quick search just now I found [this example](https://github.com/argoproj/argo-cd/blob/master/docs/operator-manual/argocd-rbac-cm.yaml) which seems to confirm that interpretation:

```
# Policy rules are in the form:
#   p, subject, resource, action, object, effect
# Role definitions and bindings are in the form:
#   g, subject, inherited-subject
```